src/Mod/PartDesign/App/FeatureHelix.cpp

```diff
@@ -237,6 +248,8 @@ App::DocumentObjectExecReturn* Helix::execute()
         if (SC.State() == TopAbs_IN) {
             result.Reverse();
         }
+        
```

Comment:
Could you fix the lint warnings? Your IDE should be configured to auto-remove the trailing spaces.